modules/spo-applying-profiles.adoc

```diff
@@ -0,0 +1,180 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-seccomp.adoc
+// * security/security_profiles_operator/spo-selinux.adoc
+
+ifeval::["{context}" == "spo-seccomp"]
+:seccomp:
+:type: seccomp
+:kind: SeccompProfile
+endif::[]
+ifeval::["{context}" == "spo-selinux"]
+:selinux:
+:type: SELinux
+:kind: SelinuxProfile
+endif::[]
+
+:_content-type: PROCEDURE
+[id="spo-applying-profiles_{context}"]
+= Applying {type} profiles to a pod
+
+Create a Pod to apply one of the created profiles.
+
+ifdef::selinux[]
+For {type} profiles, the namespace must be labelled to allow link:https://kubernetes.io/docs/concepts/security/pod-security-standards/[privileged] workloads.
+
+endif::[]
+.Procedure
+
+ifdef::seccomp[]
+. Create a Pod object that defines a `securityContext`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-pod
+spec:
+  securityContext:
+    seccompProfile:
+      type: Localhost
+      localhostProfile: operator/my-namespace/profile1.json
```

Comment:
Is `profile1.json` the name of the profile?

For consistency in the existing example above, should we use `my-profile`?

Reply:
Good catch - I changed the Creating Profiles page to `profile1` to match in all places.